README.md

```diff
@@ -1 +1,44 @@
-# sniffer
+## sniffer
+
+1. Run minikube:
+
+```
+minikube start
+```
+
+2. Run Sneeffer:
+
+```
+sudo SNIFFER_CONFIG=./configuration/SnifferConfigurationFile.json ./sniffer
```

Comment:
[TODO]: We should replace the env prefix.

@slashben Do we have a new name for the sniffer?